docs/source/index.rst

```diff
@@ -22,27 +34,35 @@ Here quick guide
 
 .. toctree::
    :maxdepth: 2
+   :hidden:
    :caption: Contents:
 
    deepinv.physics
    deepinv.datasets
+   deepinv.utils
    deepinv.models
    deepinv.loss
    deepinv.optim
    deepinv.unfolded
    deepinv.sampling
-   deepinv.transform
-   deepinv.examples
+   auto_examples/index
    deepinv.notation
 
 
+Contributing
+============================
+
+The preferred way to contribute to ``deepinv`` is to fork the `main
+repository <https://github.com/deepinv/deepinv/>`__ on GitHub,
+then submit a "Pull Request" (PR).
+
 
 Lead Developers
-==================
+============================
```

Comment:
Usually, rst linters prefer that the length of the `====` is the same as the one of the text, for easier reading in text mode (in particular in docstrings).